src/TrackRowInfoVisNominalBar.js

```diff
@@ -149,8 +154,8 @@ export default function TrackRowInfoVisNominalBar(props) {
             items: [
                 { title: "Highlight Rows", icon: HIGHLIGHTER, action: () => onSearchRows(field, "nominal", hoverValue) },
                 { title: "Filter Rows", icon: FILTER, action: () => onFilterRows(field, "nominal", hoverValue) },
```

Comment:
```suggestion
                { title: "Filter Rows", icon: FILTER, action: () => onFilterRows(field, "nominal", notOneOf) },
```

I was seeing this error when using the Context Menu "Filter Rows" button

<img width="952" alt="Screen Shot 2020-04-28 at 5 51 11 PM" src="https://user-images.githubusercontent.com/7525285/80541307-ed632500-8978-11ea-92e3-257633f06c62.png">

Reply:
Thank you for finding this issue!